source/opt/set_spec_constant_default_value_pass.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "set_spec_constant_default_value_pass.h"
 
+#include <algorithm>
```

Comment:
Clang-format puts the <algorithm> in the first place. So I will keep it there then.